doc/upgrade-cluster.md

```diff
@@ -221,6 +221,32 @@ INFO[5930] error syncing cluster version, upgrade failed. failed to sync bookkee
 ...
 ```
 
+### Handling Upgrade from one version to another
+
+When upgrading from version X to version Y (where both X and Y < 0.7), we need to ensure that we do not modify any field other than the version field while triggering the upgrade. Altering of any other parameter can be handled either before triggering the upgrade or after the upgrade has successfully completed. Same holds true even when the both X and Y >= 0.7.
```

Comment:
Removing the X and Y would make it more readable.
Lets explain it like " To trigger an upgrade please edit **only** the fields mentioned in the upgrade documentation. We do not recommend clubbing other edits ( for performance or scaling or anything else) with the upgrade trigger edit. Those can be done **prior to** or **after** upgrade has completed."